<launch>

  <!-- 
	Detection node 
  -->
  <node pkg="botello_aruco_gridboard" type="botello_aruco_gridboard_node" name="botello_aruco_gridboard_node" output="screen" >
    <rosparam command="load" file="$(find botello_aruco_gridboard)/config/botello_aruco_gridboard.yaml" /> 

    <param name="board_path" value="$(find botello_aruco_gridboard)/config/botello_layout.yaml" />
    <param name="detector_param_path" value="$(find botello_aruco_gridboard)/config/detector_params.yaml" />

    <remap from="/botello_aruco_gridboard/camera_info" to="/tello/camera/camera_info"/>
    <remap from="/botello_aruco_gridboard/image_raw" to="/tello/camera/image_raw"/>
  </node>

</launch>
